xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+     <para>
+      Replace <replaceable>LOCAL_NODE_ID</replaceable> with the ID of the local
+      node from the <filename>verb_hosts</filename> file.
+     </para>
+     </listitem>
+     <listitem>
+      <para>
+       Kill the DHCP services running on the old deployer:
+      </para>
+      <screen>(old-deployer)> sudo pkill dhcpd</screen>
+     </listitem>
+    </itemizedlist>
+   </listitem>
+  </itemizedlist>
+ </step>
+ <step>
+  <para>
+   Mount the <filename>~/sles12sp3.iso</filename> image to make it available to the HOS installer.
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Ensure that <filename>~/sles12sp3.iso</filename> was either transferred
+     from the original deployer or is downloaded to the correct location under
+     the <filename>sles12sp3.iso</filename>:
+    </para>
+    <screen>(seed)> sudo mkdir -p /media/cdrom
+(seed)> sudo mount ~/sles12sp3.iso /media/cdrom
+(seed)> sudo vi /etc/zypp/repos.d/SLES12-SP3-12.3-0.repo</screen>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Update the <literal>baseurl</literal> line to read
+      <literal>baseurl=file:///media/cdrom</literal>.
+     </para>
+    </listitem>
+   </itemizedlist>
+   </listitem>
+  </itemizedlist>
+ </step>
+<step>
+  <para>
+   Install HOS 5.SCP on the new deployer by downloading and unpacking HOS 5.SCP installation content:
+  </para>
+  <screen>(seed)> cd ~; tar -xf hos-5.1.0-*.tar
+(seed)> cd ~/hos-5.1.0; ./hos-init.bash</screen>
+ </step>
+ <step>
+  <para>
+   Prepare for running playbooks:
+  </para>
+  <screen>(seed)> cd ~/helion/hos/ansible
+(seed)> ansible-playbook -i hosts/localhost config-processor-run.yml
+(seed)> ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
+ </step>
+ <step>
+  <para>
+   Initialize ansible host facts and ensure that the existing cloud is fully
+   addressable by the new deployer:
+  </para>
+  <screen>(seed)> cd ~/scratch/ansible/next/hos/ansible
+(seed)> ansible-playbook -i hosts/verb_hosts hlm-refresh-facts.yml</screen>
+ </step>
+ <step>
+  <para>
+   Prune the primary database to remove data that is no longer relevant:
+  </para>
+<screen>(seed)> cd ~/scratch/ansible/next/hos/ansible
+(seed)> ansible-playbook -i hosts/verb_hosts percona-prune.yml</screen>
+ </step>
+</procedure>
+<procedure>
+<title>Migration Phase</title>
+
+<warning>
+ <title>Downtime</title>
+ <para>
+  Downtime will be in effect until all the following steps are completed.
+ </para>
+</warning>
+
+ <step>
+  <para>
+   Stop or pause &ostack; services on the control plane (excluding the seeds
+   that have been freshly imaged):
+  </para>
+  <screen>(seed)> cd ~/scratch/ansible/next/hos/ansible
+(seed)> ./hlm-quiesce.sh --limit '!<replaceable>SEED_HOST</replaceable>'</screen>
+  <para>
+   The <replaceable>SEED_HOST</replaceable> name must match exactly the host name in the
+   <filename>hosts/verb_hosts</filename> inventory, such as <literal>helion-cp1-c1-m1</literal>.
```

Comment:
Not sure if it is me, but "inventory" seems to be used in a jargon-y way here. Maybe replace with "...the host name that is referenced in hosts/verbs_hosts, such as ..."